core/controllers/library.py

```diff
@@ -117,18 +117,22 @@ def get(self):
                 'activity_summary_dicts': recently_published_summary_dicts,
                 'categories': [],
                 'header': feconf.LIBRARY_CATEGORY_RECENTLY_PUBLISHED,
+                'show_view_all_button': True,
             })
         if top_rated_activity_summary_dicts:
             summary_dicts_by_category.insert(0, {
                 'activity_summary_dicts': top_rated_activity_summary_dicts,
                 'categories': [],
                 'header': feconf.LIBRARY_CATEGORY_TOP_RATED_EXPLORATIONS,
+                'show_view_all_button': True,
             })
         if featured_activity_summary_dicts:
             summary_dicts_by_category.insert(0, {
                 'activity_summary_dicts': featured_activity_summary_dicts,
                 'categories': [],
-                'header': feconf.LIBRARY_CATEGORY_FEATURED_ACTIVITIES,
+                'headerI18nId':
```

Comment:
Should be header_i18n_id.

Also shouldn't this key be the same for all the groups? Each group should have a consistent representation.